src/ResourceManager/Resources/AzureRM.Resources.psd1

```diff
@@ -55,7 +55,6 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.2.0'; }
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Management.ResourceManager.dll', 
-               '.\Microsoft.Azure.ResourceManager.dll', 
```

Comment:
Are we sure this isn't needed by either Commands.Resources or Commands.ResurceManager.Cmdlets?

Reply:
How would the build (and tests) succeed otherwise? If it is needed, **something** should fail. Otherwise, we got a big gap in our build validation. Additionally, I've built each solution for this PR in a _clean_ repo, and re-ran every test in playback mode. If something doesn't fail after all of that, I have to conclude it isn't required.